migrate/20241204_vm_enforce_vcpus_memory.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+Sequel.migration do
```

Comment:
commit message typo: `memroy_gib`  in ` Enforce vcpus & memroy_gib to be set for Vm. `